src/components/CurrentCityForecast/components/HourlyForecastChart/HourlyForecastChart.jsx

```diff
@@ -0,0 +1,54 @@
+import { Line } from 'react-chartjs-2';
+import hourlyForecastData from './mockData';
+
+function HourlyForecastChart() {
+  const { hours, temperatures } = prepareChartData(hourlyForecastData);
```

Comment:
It seems like all that is happening here inside this function have nothing to do with this component's data and totally static AND should be put outside of it. Everything from line 5 to line 45 can be moved to utilities folder

Reply:
I agree with your remarks. Done.